litmus-portal/graphql-server/pkg/graphql/mutations/mutation.go

```diff
@@ -228,6 +228,31 @@ func CreateChaosWorkflow(input *model.ChaosWorkFlowInput, r store.StateData) (*m
 	}, nil
 }
 
+func UpdateWorkflow(workflow *model.ChaosWorkflowUpdateInput, r store.StateData) (*model.ChaosWorkFlowResponse, error) {
+	query := bson.D{{"cluster_id", workflow.WorkflowID}}
```

Comment:
It should be `workflow_id` in the query?

Reply:
Ohh, it is changed now